lib/unlocked/ManageBoards.dart

```diff
@@ -0,0 +1,844 @@
+
+import 'package:bubble_tab_indicator/bubble_tab_indicator.dart';
+
+class ManageBoardsScreen extends StatefulWidget {
+  const ManageBoardsScreen({Key key}) : super(key: key);
+  @override
+  _ManageBoardsScreenState createState() => _ManageBoardsScreenState();
+}
+
+class _ManageBoardsScreenState extends State<ManageBoardsScreen>
+    with SingleTickerProviderStateMixin {
+  TabController tabController;
+
+  @override
+  void initState() {
+    super.initState();
+    tabController = TabController(length: 2, vsync: this);
+  }
+
+  @override
+  void dispose() {
+    tabController.dispose();
+    super.dispose();
+  }
+
+  final List<List<String>> names = <List<String>>[
+    [
+      'assets/images/user.png',
+      'title',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+    [
+      'assets/images/user.png',
+      'title',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+    [
+      'assets/images/user.png',
+      'title',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+    [
+      'assets/images/user.png',
+      'title',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+    [
+      'assets/images/user.png',
+      'title',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+  ];
+  final List<List<String>> names2 = <List<String>>[
+    [
+      'assets/images/user.png',
+      'Public Board Tile ',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+    [
+      'assets/images/user.png',
+      'Board Tile',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+    [
+      'assets/images/user.png',
+      'Board Tile 2',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+    [
+      'assets/images/user.png',
+      'Board Tile 3',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+    [
+      'assets/images/user.png',
+      'title',
+      'number of tiles',
+      'made on',
+      'offline'
+    ],
+  ];
+  TabBar _getTabBar() {
+    return TabBar(
+      indicator: BubbleTabIndicator(
+        tabBarIndicatorSize: TabBarIndicatorSize.tab,
+        indicatorRadius: 20,
+        indicatorHeight: 60.0,
+        indicatorColor: paua,
+      ),
+      labelColor: Colors.white,
+      unselectedLabelColor: studio,
+      tabs: <Widget>[
+        Text('My Boards'),
+        Text('Community'),
+      ],
+      controller: tabController,
+    );
+  }
+
+  TabBarView _getTabBarView(tabs) {
+    return TabBarView(
+      children: tabs,
+      controller: tabController,
+    );
+  }
+
+  String _dropDownValue;
+  @override
+  Widget build(BuildContext context) {
+    final Size screenSize = MediaQuery.of(context).size;
+
+    return Scaffold(
+      appBar: PreferredSize(
+        preferredSize: Size.fromHeight(screenSize.height) / 15,
+        child: AppBar(
+          leading: GestureDetector(
```

Comment:
I think you may not need extra icon as `Navigator.push()` already implement this you call it